ibm_catalog.json

```diff
@@ -0,0 +1,185 @@
+      ],
+      "keywords": [
+        "terraform",
+        "enterprise",
+        "enterprise account",
+        "IaC",
+        "infrastructure as code",
+        "terraform",
+        "solution"
+      ],
+      "short_description": "Creates an IBM enterprise account and invite users to the account.",
+      "long_description": "This deployable architecture automates the creation of an IBM enterprise account under a parent IBM enterprise and invite users to the new account. Optionally it also creates an account group in order to organize and manage accounts hierarchically and provisions creation of multiple access groups to be assigned to the newly invited users. For more details on IBM enterprise, [see here](https://cloud.ibm.com/docs/enterprise-management?topic=enterprise-management-what-is-enterprise).",
+      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-enterprise/blob/main/solutions/fully-configurable/README.md",
+      "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-enterprise/main/images/enterprise.svg",
+      "provider_name": "IBM",
+      "features": [
+        {
+          "title": "Account and Account Group",
+          "description": "Creates account under parent enterprise account. Optionally provides ability to create account group."
+        },
+        {
+          "title": "Trusted Profile",
+          "description": "Creates trusted profile to be assumed by the account service ID to invite users."
+        },
+        {
+          "title": "Access Groups",
+          "description": "Creates multiple access groups under the new account and adds new users to it."
+        },
+        {
+          "title": "Invite Users",
+          "description": "Sends invites to new users for joining the new enterprise account."
+        }
+      ],
+      "support_details": "This product is in the community registry, as such support is handled through the originated repo. If you experience issues please open an issue in that repository [https://github.com/terraform-ibm-modules/terraform-ibm-enterprise/issues](https://github.com/terraform-ibm-modules/terraform-ibm-enterprise/issues). Please note this product is not supported via the IBM Cloud Support Center.",
+      "flavors": [
+        {
+          "label": "Fully configurable",
+          "name": "fully-configurable",
+          "install_type": "fullstack",
+          "working_directory": "solutions/fully-configurable",
+          "iam_permissions": [
+            {
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::role:Administrator"
+              ],
+              "service_name": "enterprise",
+              "notes": "Required when you are creating account and account group under an IBM Enterprise."
+            }
+          ],
+          "configuration": [
+            {
+              "key": "ibmcloud_api_key"
+            },
+            {
+              "key": "region",
```

Comment:
There is no regional service deployed in this DA. It doesn't need a region input. You are only using it in the provider config, so just remove it from there